FordLabs/retroquest
# Migrate to Vendor Netural Database Version Control fixes #268

- Implement Liquibase database version control
- Remove flyway database version control
- Switch Docker image from MariaDB to Postgresql

## Overview
This PR moves database versioning from Flyway to Liquibase.  Allowing for the application to be database vendor neutral. 

### Notes
- Fixes #268 
- user and user_mapping_table appear to be unused

## Testing Instructions
1. Running with H2 database - Application should start up initialize all tables and work properly
2. Running with new postgresql database - Application should start up initialize all tables and work properly
3. Running with existing database - Application should start up and skip all table creation

## Implementation Notes:
After deploying to production, you can delete the schemaversion tables created by flway.
